docs/user-guide/logs/pipeline-config.md

```diff
@@ -326,6 +326,125 @@ In the above example, the configuration of the `csv` processor includes the foll
 - `trim`: Whether to trim whitespace. Defaults to `false`.
 - `ignore_missing`: Ignores the case when the field is missing. Defaults to `false`. If the field is missing and this configuration is set to `false`, an exception will be thrown.
 
+### `json_path` (experimental)
+
+Note: The `json_path` processor is currently in the experimental stage and may be subject to change.
+
+The `json_path` processor is used to extract fields from JSON data. Here's an example configuration:
+
+```yaml
+processors:
+  - json_path:
+      fields:
+        - complex_object
+      json_path: "$.shop.orders[?(@.active)].id"
+      ignore_missing: true
+      result_index: 1
+```
+
+In the above example, the configuration of the `json_path` processor includes the following fields:
+
+- `fields`: A list of field names to be extracted.
+- `json_path`: The JSON path to extract.
+- `ignore_missing`: Ignores the case when the field is missing. Defaults to `false`. If the field is missing and this configuration is set to `false`, an exception will be thrown.
+- `result_index`: The index of the result to extract. Defaults to all results. because the result is an array.
+
+#### JSON path syntax
+
+The JSON path syntax is based on the [jsonpath-rust](https://github.com/besok/jsonpath-rust) library.
+
+At this stage we only recommend using some simple field extraction operations to facilitate the extraction of nested fields to the top level.
+
+#### `json_path` example
+
+For example, given the following log data:
+
+```json
+{
+  "product_object": {
+    "hello": "world"
+  },
+  "product_array": [
+    "hello",
+    "world"
+  ],
+  "complex_object": {
+    "shop": {
+      "orders": [
+        {
+          "id": 1,
+          "active": true
+        },
+        {
+          "id": 2
+        },
+        {
+          "id": 3
+        },
+        {
+          "id": 4,
+          "active": true
+        }
+      ]
+    }
+  }
+}
+```
+
+Using the following configuration:
+
+```yaml
+processors:
+  - json_path:
+      fields:
+        - product_object, object_target
+      json_path: "$.hello"
+      result_index: 0
+  - json_path:
+      fields:
+        - product_array, array_target
+      json_path: "$.[1]"
+      result_index: 0
+  - json_path:
+      fields:
+        - complex_object, complex_target_1
```

Comment:
The `complex_target_1` field isn't in the JSON example.
If two values ​​are separated by `,` to indicate renaming, add a description in `field`.
Please also explain if the `field` in other processors have this feature. If it is, then please modify all the `field` description.